test/coreputation/is_slashed.js

```diff
@@ -17,31 +17,34 @@
 const BN = require('bn.js');
 const { AccountProvider } = require('../test_lib/utils.js');
 
-const CoReputation = artifacts.require('CoreputationTest');
+const Coreputation = artifacts.require('CoreputationTest');
 
-contract('Coreputation::getReputation', (accounts) => {
+contract('Coreputation::isSlashed', (accounts) => {
   let accountProvider;
-  let coReputation;
+  let coreputationInstance;
   let validatorInfo;
+  let coconsensus;
 
   beforeEach(async () => {
     accountProvider = new AccountProvider(accounts);
-    coReputation = await CoReputation.new();
+    coconsensus = accountProvider.get();
+    coreputationInstance = await Coreputation.new(coconsensus);
     validatorInfo = {
       validator: accountProvider.get(),
       reputation: new BN('10'),
     };
-    await coReputation.upsertValidator(
+    await coreputationInstance.upsertValidator(
       validatorInfo.validator,
       validatorInfo.reputation,
+      { from: coconsensus },
     );
-    await coReputation.setValidatorSlashed(
+    await coreputationInstance.setValidatorSlashed(
       validatorInfo.validator,
     );
   });
 
   it('should return correct reputation for a known validator', async () => {
```

Comment:
this test description does not match the test

Reply:
Test description is updated